### PR TITLE
Cut the mustard

### LINKED ISSUE #5: Add "cut the mustard" test
JS will fail in browsers not supporting `classList` etc.
Only execute script in capable browsers. Give others a fallback.

### PR DESCRIPTION
**Pull request by reinoute**
Resolves #5

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -28,56 +44,66 @@
...
+
+            <button type="submit" class="button" tabindex="-1">GO</button>
+        </form>
+
+    </div>
 </header>
 
-<iframe src="https://voorhoede.nl" width="100%" height="100%"></iframe>
+<div class="update-browser">
+    Accessibility Viewer requires a browser that supports SVG filters and classList. If you can, <a
```
Comment (jbmoelker) [inline src/index.html]: Rephrase? "Your browser is unable to emulate impairments. If you can, update your browser to uses the Accessibility Viewer." We don't have to get technical on our users.

File: src/index.html [hunk 2]
```diff
@@ -28,56 +44,66 @@
...
+        <span class="click-stealer-toggler">
+            <input type="checkbox" id="click-stealer-toggler" tabindex="-1">
+            <label class="label-text" for="click-stealer-toggler">
+                Keyboard only
+            </label>
+        </span>
+
+        <form class="url-navigator">
+            <label for="url">
+                <span class="a11y-sr-only">What page do you want to view?</span>
```
Comment (jbmoelker) [inline src/index.html]: Rephrase? "What page (url) do you ..."

File: src/index.html [hunk 3]
```diff
@@ -28,56 +44,66 @@
...
-    <form class="url-navigator">
-        <label for="url">
-            <span class="a11y-sr-only">What page do you want to view?</span>
-            <input type="text" id="url" placeholder="Enter url" tabindex="-1">
+        <label class="module-selector-label" for="color-blindness-selector">
+            <span class="a11y-sr-only">Select level of color blindness</span>
+            <select class="module-selector-select" id="color-blindness-selector" tabindex="-1">
+                <optgroup label="Color blindness">
+                    <option value="normal">No color blindness</option>
+                    <option value="protanopia">Protanopia(no red)</option>
```
Comment (jbmoelker) [inline src/index.html]: add space behind names:

```
Protanopia (no red)
          ^
```

File: src/index.html [hunk 4]
```diff
@@ -4,17 +4,33 @@
...
     <meta name="description" content="Emulate several forms of visual imparity for developing better accessible websites">
     <meta name="keywords" content="a11y, emulator, visual imparity, color blindness">
 
     <link rel="stylesheet" href="index.css" />
+
+    <script>
+        (function () {
+            var SUPPORTS_CLASSLIST = ('classList' in document.body);
+            // borrowed from https://github.com/Modernizr/Modernizr/blob/master/feature-detects/svg/filters.js
+            var SUPPORTS_SVG_FILTERS = 'SVGFEColorMatrixElement' in window &&
```
Comment (jbmoelker) [inline src/index.html]: Put `(` ... `)` around it like you do with `SUPPORTS_CLASSLIST`.

File: src/index.html [hunk 5]
```diff
@@ -4,17 +4,33 @@
...
+                    SVGFEColorMatrixElement.SVG_FECOLORMATRIX_TYPE_SATURATE == 2;
+
+            // only enhance capable browsers
```
Comment (jbmoelker) [inline src/index.html]: You could replace this comment with code:

``` javascript
var IS_CAPABLE_BROWSER = (SUPPORTS_CLASSLIST && SUPPORTS_SVG_FILTERS);
```

File: src/index.html [hunk 6]
```diff
@@ -4,17 +4,33 @@
...
+            if (SUPPORTS_CLASSLIST && SUPPORTS_SVG_FILTERS) {
+                document.body.className += ' is-enhanced';
```
Comment (jbmoelker) [inline src/index.html]: You have just established `classList` is supported. So change to:

``` javascript
document.body.classList.add('is-enhanced');
```

**Comment by jbmoelker** [comment]
add `[lang="en"]` (no strictly part of this PR, but while you are at it :) )

File: src/index.html (hunk 1 above)
Comment (reinoute) [inline src/index.html]: Done

File: src/index.html (hunk 2 above)
Comment (reinoute) [inline src/index.html]: done

File: src/index.html (hunk 3 above)
Comment (reinoute) [inline src/index.html]: Done

File: src/index.html (hunk 5 above)
Comment (reinoute) [inline src/index.html]: Done

File: src/index.html (hunk 6 above)
Comment (reinoute) [inline src/index.html]: This is shorter? Changed it anyway!

**Pull request merged**